### ISSUE TITLE
Multilingual names

### ISSUE THREAD

**Issue by pyrog** [osmlab/name-suggestion-index#4673]
Some templates are duplicated in some countries (2 or more official languages).

Examples of post boxes:
* [Algeria](https://nsi.guide/index.html?t=operators&k=amenity&v=post_office&tt=Q2836029)
* [Canada](https://nsi.guide/index.html?t=operators&k=amenity&v=post_box&tt=Q1032001&cc=ca)
* [Greece](https://nsi.guide/index.html?t=operators&k=amenity&v=post_office&tt=Q1275135) ?
* [Marocco](https://nsi.guide/index.html?t=operators&k=amenity&v=post_box&tt=Q3399778)
* [Switzerland](https://nsi.guide/index.html?t=operators&k=amenity&v=post_box&tt=Q614803)

Car sharing:
* [Belgium](https://nsi.guide/index.html?t=operators&k=amenity&v=car_sharing&tt=cambio&cc=be)
* [Switzerland](https://nsi.guide/index.html?t=operators&k=amenity&v=car_sharing&cc=ch)

This issue could be solved by moving NSI templates in OSM Wikibase (cf. #4657)

**Comment by bhousel** [comment]
The reason we have a lot of duplicates in the `operators` tree is because in the collect_all script, the threshold for collecting an `operator` (10 uses) is much lower than the threshold for collecting a `brand` (50 uses).  

We can remove the duplicates manually, or disambiguate by location.. For example the Canada Post you linked to above is set up as "Postes Canada" in Quebec and "Canada Post" elsewhere.  This is correct.

**Comment by bhousel** [comment]
I think I got all these duplicates removed..

**Comment by pyrog** [comment]
thank you 😃

**Comment by pyrog** [comment]
Oups, seem [Canada](https://nsi.guide/index.html?t=operators&k=amenity&v=post_box&tt=Q1032001&cc=ca) is still duplicated.

And for [Switzerland](https://nsi.guide/index.html?t=operators&k=amenity&v=post_box&tt=Q614803) NSI seem to impose name in german.
I think that the choice should be left to the user. In french speaking "cantons" they use only the french name.
And in german speaking, only the german.
Some cantons or towns use both german and french for official names.

In Romansh language, the name is "La Posta".

**Comment by Identitaet** [comment]
We'd probably need a custom file for the swiss language borders

**Comment by bhousel** [comment]
> Oups, seem [Canada](https://nsi.guide/index.html?t=operators&k=amenity&v=post_box&tt=Q1032001&cc=ca) is still duplicated.

Yeah this is intentional.  The POIs will be in French in Quebec, English elsewhere.  (#3162 has more on this)


> And for [Switzerland](https://nsi.guide/index.html?t=operators&k=amenity&v=post_box&tt=Q614803) NSI seem to impose name in german.
> I think that the choice should be left to the user. In french speaking "cantons" they use only the french name.
> And in german speaking, only the german.
> Some cantons or towns use both german and french for official names.

Yeah I just set all the names that are in use for Die Post. Their website seems to show that they use a very similar branding in all languages.  There is some precedent in the index here on #3162:  "For some entries we already do add multiple language tags where multiple local languages are commonly spoken (e.g. things in Belarus commonly use both Russian and Belarusian)"

Basically if a POI appears in different languages in a single country, the decision tree looks like:
- if they languages are geographically distinct in a simple way (like Quebec), especially if branding is really different in different places (like KFC/PFK), we make multiple items for them scoped to where those languages/branding are used.
- if the languages are used everywhere (like Belarus), especially if branding is really similar everywhere (like Белпошта/Белпочта) we make a single item and add multilingual tags.

If people aren't ok with either of those options, (every POI is really different and mappers won't accept a bunch of multilingual tags) it might be better to just remove it from the index.
